src/app/courses/progress-courses/courses-progress-leader.component.ts

```diff
@@ -48,6 +49,7 @@ export class CoursesProgressLeaderComponent implements OnInit, OnDestroy {
       this.filterSubmittedExamSteps(submissions);
       this.dialogsLoadingService.stop();
     });
+    this.chartLabel = 'Steps';
```

Comment:
You can just initialize the value to `'Steps'` on line 19 rather than adding this here.

Reply:
Got it